components/global_header/left_controls/history_buttons/history_buttons.tsx

```diff
@@ -62,7 +62,7 @@ const HistoryButtons = (): JSX.Element => {
         );
     };
 
-    const handleButtonMessage = (message: {origin: string; data: {type: string; message: {enableBack: boolean; enableForward: boolean}}}) => {
+    const handleButtonMessage = useCallback((message: {origin: string; data: {type: string; message: {enableBack: boolean; enableForward: boolean}}}) => {
```

Comment:
Just curious, how come we don't format the logic the same way as above? (in which we place the function within the `useEffect` call)

Reply:
I think I prefer the first way of doing this since it keeps everything contained in the `useEffect`. They do seem to do the same thing though

Reply:
Actually, both options are good options, I don't know exactly why I did this differently.